### PR TITLE
Plugins docstrings and formatting

### PR DESCRIPTION
**Pull request by lakshyacis**
Minor changes in plugins

### CODE REVIEW

File: src/plugins/sound_gstreamer.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/sound_gstreamer.py
```
Comment (PeterSurda) [inline src/plugins/sound_gstreamer.py]: maybe just `Sound plugin` too, it probably just uses a different backend

File: src/plugins/sound_playfile.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/sound_playfile.py
```
Comment (PeterSurda) [inline src/plugins/sound_playfile.py]: maybe just `Sound plugin` too, it probably just uses a different backend

File: src/plugins/notification_notify2.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/notification_notify2.py
-===================================
+Plugin notifies new message
```
Comment (g1itch) [inline src/plugins/notification_notify2.py]: Notification plugin using notify2

File: src/plugins/indicator_libmessaging.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/indicator_libmessaging.py
-=====================================
+Libmessageindicator plugin
```
Comment (g1itch) [inline src/plugins/indicator_libmessaging.py]: Indicator plugin using libmessaging

File: src/plugins/sound_canberra.py [hunk 5]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/proxyconfig_stem.py
-===================================
+Sound plugin
```
Comment (g1itch) [inline src/plugins/sound_canberra.py]: Sound theme plugin using pycanberra

File: src/plugins/sound_gstreamer.py (hunk 1 above)
Comment (g1itch) [reply src/plugins/sound_gstreamer.py]: Sound notification plugin using gstreamer

File: src/plugins/sound_playfile.py (hunk 2 above)
Comment (g1itch) [reply src/plugins/sound_playfile.py]: Sound notification plugin using winsound

File: src/plugins/sound_canberra.py (hunk 5 above)
Comment (lakshyacis) [reply src/plugins/sound_canberra.py]: Thank you

**Comment by PeterSurda** [comment]
@g1itch any more objections?

**Comment by g1itch** [comment]
> @g1itch any more objections?

No

**Pull request merged**